pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -1081,3 +1083,101 @@ def get_volume_of_n_sphere_in_p_norm(n=3, p=2):
     """
     return (2*gamma(1+1/p))**n/gamma(1+n/p)
 
+
+class NeighborsTraj(DataContainer):
```

Comment:
```suggestion
class NeighborsTrajectory(DataContainer):
```
I'd actually prefer to spell it out, since users don't have to type it.